iib/workers/tasks/konflux_utils.py

```diff
@@ -0,0 +1,331 @@
+    target_cluster_url = getattr(worker_config, 'iib_konflux_cluster_url', None)
+    target_cluster_token = getattr(worker_config, 'iib_konflux_cluster_token', None)
+    target_cluster_ca_cert = getattr(worker_config, 'iib_konflux_cluster_ca_cert', None)
+
+    # If no Konflux configuration is provided, raise an error
+    if not target_cluster_url or not target_cluster_token or not target_cluster_ca_cert:
+        raise IIBError(
+            "Konflux configuration is required. Please set "
+            "iib_konflux_cluster_url, iib_konflux_cluster_token, and "
+            "iib_konflux_cluster_ca_cert in IIB worker configuration."
+        )
+
+    log.info(f"Configuring Kubernetes client for cross-cluster access to {target_cluster_url}")
+
+    configuration = _create_kubernetes_configuration(
+        target_cluster_url, target_cluster_token, target_cluster_ca_cert
+    )
+
+    return client.CustomObjectsApi(client.ApiClient(configuration))
+
+
+def _create_kubernetes_configuration(url, token, ca_cert):
+    """
+    Create Kubernetes configuration with authentication and SSL settings.
+
+    :param str url: The Kubernetes cluster API URL
+    :param str token: The authentication token for the cluster
+    :param str ca_cert: The CA certificate for SSL verification (file path or content)
+    :return: Configured Kubernetes Configuration object
+    """
+    configuration = client.Configuration()
+    configuration.host = url
+    configuration.api_key_prefix['authorization'] = 'Bearer'
+    configuration.api_key['authorization'] = token
+
+    # If CA cert is provided as a string, write it to a temp file
+    if not os.path.isfile(ca_cert):
+        import tempfile
+
+        with tempfile.NamedTemporaryFile(mode='w', delete=False, suffix='.crt') as f:
+            f.write(ca_cert)
+            ca_cert = f.name
+
+    configuration.ssl_ca_cert = ca_cert
+    return configuration
+
+
+def find_pipelinerun(commit_sha: str) -> List[Dict[str, Any]]:
+    """
+    Find the Konflux pipelinerun triggered by the git commit.
+
+    :param str commit_sha: The git commit SHA to search for
+    :return: List of pipelinerun objects matching the commit SHA
+    :rtype: List[Dict[str, Any]]
+    :raises IIBError: If there's an error fetching pipelineruns
+    """
+    try:
+        log.info(f"Searching for pipelineruns with commit SHA: {commit_sha}")
+
+        v1_client = _get_kubernetes_client()
+        worker_config = get_worker_config()
+        namespace = getattr(worker_config, 'iib_konflux_namespace', 'iib-tenant')
+
+        runs = v1_client.list_namespaced_custom_object(
+            group="tekton.dev",
+            version="v1",
+            namespace=namespace,
+            plural="pipelineruns",
+            label_selector=f"pipelinesascode.tekton.dev/sha={commit_sha}",
+        )
+
+        items = runs.get("items", [])
+        log.info(f"Found {len(items)} pipelinerun(s) for commit {commit_sha}")
```

Comment:
I was looking into other modules of IIB's code and we use the following format for logging:

```suggestion
        log.info(f"Found %s pipelinerun(s) for commit %s", len(items), commit_sha)
```

Can you please update all logging functions to behave the same?